assets/ts/stop/components/amenities/AccessibilityAmenityCard.tsx

```diff
@@ -23,7 +23,7 @@ const accessibilityNames: {
 
 const StationFeatures = (features: AccessibilityType[]): JSX.Element => (
   <>
-    <h2 className="text-2xl">Station Features</h2>
```

Comment:
Curiosity: Did you remove this because it wasn't necessary (and is kind of antipattern-y)? Or because it was wrong?

Reply:
Okay - this _does_ change the font size (makes it a bit smaller). Is that intentional as part of this work?